### DISCUSSION TITLE
Search cache not working, shows no results

### DISCUSSION THREAD

**Discussion by paolofacchinetti** [requarks/wiki#5257] [Error / Bug Report]
Whenever I search for something (and see my results), then exit the search and then search once more the same exact keyword, Wiki.js shows no results.
Even just searching for a word (i.e. "hello"), then deleting the word and typing it once more, gives zero results.

Note that this only works when i search the same exact keyword letter by letter. Even adding an additional space makes the search function correctly. On a hunch I believe it has to do with caching of the search results?

Refreshing the page with F5 fixes the issue.

Some info on my installation:
- wiki.js 2.5.279
- postgreSQL 11.15
- docker-compose installation
- I'm using the search engine setting "Database - PostgreSQL"

**Comment by paolofacchinetti** [comment]
I created an issue based on this discussion, but looks like it was randomly deleted without explanation?

**Reply by NGPixel** [reply]
Because here is the correct place to report bugs / questions, not via Issues.
